torchvision/prototype/transforms/_temporal.py

```diff
@@ -0,0 +1,19 @@
+from typing import Any, Dict
+
+from torchvision.prototype import features
+from torchvision.prototype.transforms import functional as F, Transform
+
+
+class UniformTemporalSubsample(Transform):
+    _transformed_types = (
+        features.is_simple_tensor,
+        features.Video,
```

Comment:
```suggestion
        features.Video
```

to prevent `black` from exploding this tuple